.github/workflows/notify-slack.yml

```diff
@@ -0,0 +1,67 @@
+name: Notify Applications team on "Ready for review" PRs
+
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  notify-slack:
+    if: github.event.label.name == 'Ready for review'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Send notification to Slack
+        run: |
+          SLACK_WEBHOOK_URL="${{ secrets.SLACK_WEBHOOK_URL }}"
+          PR_NR="${{ github.event.pull_request.number }}"
+          PR_URL="${{ github.event.pull_request.html_url }}"
+          PR_TITLE="${{ github.event.pull_request.title }}"
+          PR_AUTHOR="${{ github.event.pull_request.user.login }}"
+          PR_ADDITIONS="${{ github.event.pull_request.additions }}"
+          PR_DELETIONS="${{ github.event.pull_request.deletions }}"
+          REPO_NAME="${{ github.event.repository.name }}"
+
+          curl -X POST -H 'Content-type: application/json' \
+          --data "{
+            \"text\": \"*Pull Request Ready for Review!*\", 
+            \"blocks\": [
+              {
+                \"type\": \"section\",
+                \"text\": {
+                  \"type\": \"mrkdwn\",
+                  \"text\": \"@applications-team: :rocket: PR by ${PR_AUTHOR} needs a review: \`+${PR_ADDITIONS} -${PR_DELETIONS}\` <${PR_URL}|${REPO_NAME}#${PR_NR}: ${PR_TITLE}>\"
+                }
+              }
+            ]
+          }" \
+          $SLACK_WEBHOOK_URL
+
+  remove-label-on-approval:
+    if: github.event.review.state == 'approved'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Remove label from PR
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const { data: pullRequest } = await github.rest.pulls.get({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: context.issue.number
+            });
+
+            const labelToRemove = 'Ready for review';
+            const labels = pullRequest.labels.map(label => label.name);
+
+            if (labels.includes(labelToRemove)) {
+              await github.rest.issues.removeLabel({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                issue_number: context.issue.number,
+                name: labelToRemove
+              });
+              console.log(`Removed label: ${labelToRemove}`);
```

Comment:
Are these console.log useful?

Reply:
Can you help me check it by approving the PR :smile: ? I havent been able to confirm this workflow action yet (cant approve my own PR) The console log should appear in the repos Actions tab but not 100% sure yet

Reply:
hmm this action didnt work, will need to revisit it :frowning_face:

Reply:
confirmed that logs appear in the Actions tab :+1: : 
![image](https://github.com/user-attachments/assets/6e6e3f2e-745f-4c5d-aa72-6aba0b70d61c)